packages/react-tabster/src/TabsterProvider.tsx

```diff
@@ -37,9 +41,9 @@ export const useTabsterProvider = (props: TabsterProviderProps, ref: React.Ref<H
   const defaultView = state.document?.defaultView || undefined;
   const tabsterOptions = state.customRoot ? {} : { autoRoot: {} };
 
-  state.contextValue = defaultView && (getCurrentTabster(defaultView) ?? createTabster(defaultView, tabsterOptions));
-
-  return state;
+  return assign(state, {
+    contextValue: defaultView && (getCurrentTabster(defaultView) ?? createTabster(defaultView, tabsterOptions)),
+  });
```

Comment:
Can you please clarify why we need this change? Is there any difference between previous code?